ui-tests/tests/jupytereverywhere.spec.ts

```diff
@@ -280,3 +280,31 @@ test('Should remove View Only banner when the Create Copy button is clicked', as
     page.locator('.jp-NotebookPanel-toolbar [data-jp-item-name="insert"]')
   ).toBeVisible();
 });
+
+test.describe('Landing page', () => {
+  test('Should render the landing page as expected', async ({ page }) => {
+    await page.goto('index.html');
```

Comment:
Maybe?

```suggestion
    await page.goto('index.html');
    await page.waitForSelector('.je-hero');
```

Reply:
Yes, applied – let's see if it works!

Reply:
Hmm so it went to the other `index.html`:

```
      - waiting for locator('.je-hero') to be visible
        - waiting for" http://localhost:3000/lab/index.html" navigation to finish...
        - navigated to "http://localhost:3000/lab/index.html"
```

Reply:
It does happen for me when trying to use the built artifacts. Top-level `index.html` does redirect to `lab/index.html`